.releaserc

```diff
@@ -0,0 +1,25 @@
+{
+  "plugins": [
+    ["@semantic-release/commit-analyzer", {
+      "releaseRules": [
+        {breaking: true, release: 'minor'},
+        {type: 'feat', release: 'patch'},
+      ],
```

Comment:
Once KaTeX reaches 1.0, these can be changed at that point, right?

Reply:
Yes. In fact, the default option can be used.